pkg/clients/aws.go

```diff
@@ -92,6 +98,45 @@ func LoadConfig(data []byte, profile, region string) (*aws.Config, error) {
 	return &config, err
 }
 
+// LoadSAConfig assumes an IAM role configured via a ServiceAccount.
+// https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
+func LoadSAConfig(ctx context.Context, region string) (*aws.Config, error) {
+	cfg, err := external.LoadDefaultAWSConfig()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We should wrap these errors now that there's a small handful of branches that return them.